data-raw/crosswalks/neighborhood_to_neighborhood.R

```diff
@@ -0,0 +1,11 @@
+neigh_ccc
+neigh_sna
+neigh_cchmc
+
+neighs <- mapview::mapview(neigh_ccc, col.regions = "red", legend = FALSE) +
+  mapview::mapview(neigh_sna, col.regions = "blue", legend = FALSE) +
+  mapview::mapview(neigh_cchmc, col.regions = "green", legend = FALSE)
+
+# do we expect users to convert neighborhood data to other neighborhoods? These are all
+# different, but usually only slightly... would end up with very small weights in a
+# lot of cases
```

Comment:
was this left over from our discussion about neighborhood to neighborhood mappings?   we can delete this, right?

Reply:
also can remove the `.DS_Store` files (or add to `.gitignore`)